src/staticfile/integration/integration_suite_test.go

```diff
@@ -17,27 +19,48 @@ import (
 
 var bpDir string
 var buildpackVersion string
+var buildpackZipFile string
 var packagedBuildpack cutlass.VersionedBuildpackPackage
 
 func init() {
-	flag.StringVar(&buildpackVersion, "version", "", "version to use (builds if empty)")
+	flag.StringVar(&buildpackVersion, "version", "", "version to use (builds if version and zipFile empty)")
 	flag.BoolVar(&cutlass.Cached, "cached", true, "cached buildpack")
 	flag.StringVar(&cutlass.DefaultMemory, "memory", "128M", "default memory for pushed apps")
 	flag.StringVar(&cutlass.DefaultDisk, "disk", "128M", "default disk for pushed apps")
+	flag.StringVar(&buildpackZipFile, "zipFile", "", "buildpack zip file in buildpack root dir (builds if zipFile and version empty)")
```

Comment:
Description might be clearer as "path to already built buildpack zip file (builds if zipFile and version are empty)"